be2-scala/src/test/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandlerTest.scala

```diff
@@ -0,0 +1,65 @@
+package ch.epfl.pop.pubsub.graph.handlers
+
+import akka.actor.Actor
+import akka.actor.ActorSystem
+import akka.actor.Props
+import akka.actor.typed.ActorRef
+import akka.pattern.AskableActorRef
+import akka.testkit.ImplicitSender
+import akka.testkit.TestKit
+import akka.testkit.TestProbe
+import akka.util.Timeout
+import ch.epfl.pop.pubsub.graph.DbActor
+import ch.epfl.pop.pubsub.graph.PipelineError
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.FunSuiteLike
+import org.scalatest.Matchers
+
+import scala.concurrent.duration.FiniteDuration
+import ch.epfl.pop.model.network.requests.rollCall.JsonRpcRequestCreateRollCall
+import ch.epfl.pop.pubsub.graph.validators.RpcValidator
+import ch.epfl.pop.model.network.MethodType
+import ch.epfl.pop.model.network.method.ParamsWithMessage
+import ch.epfl.pop.model.objects.Channel
+
+
+class RollCallHandlerTest extends TestKit(ActorSystem("RollCall-DB-System")) with FunSuiteLike with ImplicitSender with Matchers with BeforeAndAfterAll{
+  // Implicites for system actors
+  implicit val duration = FiniteDuration(5 ,"seconds")
+  implicit val timeout = Timeout(duration)
+
+  override def afterAll(): Unit = {
+    // Stops the testKit
+    TestKit.shutdownActorSystem(system)
+  }
+
+  def mockDb: AskableActorRef = {
```

Comment:
This method can take parameters to generate a custom DBactor with the desired behavior